docs/listings/hello2_list_verbose.txt

```diff
@@ -1,25 +1,27 @@
 Loading user configuration
 Loading configuration file: 'tutorials/config/settings.py'
-Detecting system
-Looking for a matching configuration entry for system 'host'
-Configuration found: picking system 'generic'
-Selecting subconfig for 'generic'
+Detecting system using method: 'hostname'
+Using standard hostname...
+Retrieved hostname: 'tresa.local'
+Looking for a matching configuration entry
+Configuration found: picking system 'catalina'
 Initializing runtime
-Selecting subconfig for 'generic:default'
 Initializing system partition 'default'
-Selecting subconfig for 'generic'
-Initializing system 'generic'
+Initializing system 'catalina'
 Initializing modules system 'nomod'
-detecting topology info for generic:default
-> found topology file '/home/user/.reframe/topology/generic-default/processor.json'; loading...
+detecting topology info for catalina:default
```

Comment:
This listing is not correct. I looked up where we load it and it is in tutorial_tips_tricks.rst. There we say:
"You can see all the different phases ReFrame’s frontend goes through when loading a test. The first “strange” thing to notice in this log is that ReFrame picked the generic system configuration. This happened because it couldn’t find a system entry with a matching hostname pattern. However, it did not impact the test loading, because these tests are valid for any system, ..."

Reply:
Good point. I think that this sentence should be removed, because it doesn't make sense. When we wrote this, almost 2 years ago, I was probably seeing that failure because I was running from VPN and reframe could not retrieve match the local hostname. Later on, we introduced the auto-detection from the fqdn and sometimes my mac's fqdn was rather weird, so reframe could not still pick the pattern. Both behaviours were not expected actually and that was an explanation for that listing. I will remove this now.